### ISSUE TITLE
FormatException: Scheme not starting with alphabetic character (at character 1)

### ISSUE THREAD

**Issue by nilcemar** [Flutterando/uno#27]
When use uno.get('address') the error "FormatException: Scheme not starting with alphabetic character (at character 1)" is issued because a "&" is added to adrress -> "&https://api.ipify.org"

Example:
```dart
await _uno.get(
          'https://api.ipify.org',
          headers: {HttpHeaders.contentTypeHeader: 'text/plain'},
        ).then((response) {
          if (response.toString().isNotEmpty) {
            ipAddress = response.toString();
            debugPrint('IP Address: $ipAddress');
          }         
        });
```

**Comment by erickximenes** [comment]
Good morning @nilcemar, this error has been corrected in version 1.1.2. Just update your package to:

uno: ^1.1.2

**Comment by erickximenes** [comment]
https://github.com/Flutterando/uno/issues/29
